integration/sawtooth_integration/docker/test_peer_list.yaml

```diff
@@ -127,10 +131,66 @@ services:
           --endpoint tcp://validator-4:8800 \
           --bind component:tcp://eth0:4004 \
           --bind network:tcp://eth0:8800 \
+          --bind consensus:tcp://eth0:5005 \
           --peers tcp://validator-2:8800,tcp://validator-3:8800
     \""
     stop_signal: SIGKILL
 
+  devmode:
```

Comment:
These devmode containers all have the same name.

Reply:
Ah, that will not work. I think I may drop this commit and bring these changes in another PR.